output/gigpress_shows.php

```diff
@@ -17,6 +17,7 @@ function gigpress_shows($filter = null, $content = null) {
 
 	global $wpdb, $gpo;
 	$further_where = $limit = '';
+	$some_results = FALSE;
```

Comment:
@bordoni I normally would but the entire page is using the uppercase standard. Have we decided what we are going to do with GigPress, as it doesn't use the MT standard? Are we going to try and convert the whole thing?